litmus-portal/frontend/src/utils/validate.ts

```diff
@@ -19,6 +19,15 @@ export const validateEmail = (value: string) => {
   return false;
 };
 
+export const validateUsername = (value: string) => {
+  const usernameValid = /^[a-zA-Z\d@\.]+$/;
```

Comment:
```suggestion
  const usernameValid = /^[a-zA-Z\d-_@\.]+$/;
```
My username would not be validated with the regex currently present, this suggestion adds - and _ chars to the regex which is the apt validation for a username. Could you please make this suggested change!